dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -248,7 +248,7 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
         PushNotificationRegistrationToken token = { cookie, registeredTaskFromBuilder };
         scopeExitToCleanRegistrations.release();
 
-        PushNotificationTelemetry::ActivatorRegisteredByApi(S_OK);
+        PushNotificationTelemetry::ActivatorRegisteredByApi(S_OK, details.Options());
```

Comment:
Same issue as in UnregisterActivator: we can throw but we don't log the error code.

Reply:
Also addressed